public/api/conf/1.0/schemas/subSchemas/identityOfCompanySubmitting.json

```diff
@@ -7,15 +7,15 @@
   ],
   "properties": {
     "companyName": {
-      "description": "Name of the appointing company",
+      "description": "Name of the revoking company.",
```

Comment:
This subschema is used by both appoint and revoke so making this change will break it for appoint. I think we need to duplicate this subschema and then change the descriptions for revoke